### PR TITLE
The Layer Killer

### PR DESCRIPTION
**Pull request by DreamySkrell**
kills all layer vars on every map
adds layer maplint

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

**Review by Arrow768 — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]
In the help message you should specify how the layers should be edited instead.

**Comment by DreamySkrell** [comment]
> In the help message you should specify how the layers should be edited instead.

done 

help: "Do not edit layer manually in the map, edit them in code instead."

**Review by FluffyGhoster — APPROVED** [review]

File: tools/maplint/lints/layer_varedits.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+help: "Do not edit layer manually in the map, edit them in code instead."
+"*":
+  banned_variables:
+  - layer
+  - open_layer
+  - closed_layer
```
Comment (FluffyGhoster) [inline tools/maplint/lints/layer_varedits.yml:6]: These only exist for /obj/machinery/door

**Pull request merged**